projects/tinkerbell/cluster-api-provider-tinkerbell/patches/0015-Make-hardware-template-workflow-CRDs-namespace-scope.patch

```diff
@@ -0,0 +1,173 @@
+ tink/controllers/workflow/controller.go        | 4 ++--
+ 8 files changed, 15 insertions(+), 7 deletions(-)
+
+diff --git a/config/crd/bases/tinkerbell.org_hardware.yaml b/config/crd/bases/tinkerbell.org_hardware.yaml
+index 47f5edb..e0a52a1 100644
+--- a/config/crd/bases/tinkerbell.org_hardware.yaml
++++ b/config/crd/bases/tinkerbell.org_hardware.yaml
+@@ -16,7 +16,7 @@ spec:
+     listKind: HardwareList
+     plural: hardware
+     singular: hardware
+-  scope: Cluster
++  scope: Namespaced
+   versions:
+   - name: v1alpha1
+     schema:
+diff --git a/config/crd/bases/tinkerbell.org_templates.yaml b/config/crd/bases/tinkerbell.org_templates.yaml
+index 4e0a558..c364dab 100644
+--- a/config/crd/bases/tinkerbell.org_templates.yaml
++++ b/config/crd/bases/tinkerbell.org_templates.yaml
+@@ -16,7 +16,7 @@ spec:
+     listKind: TemplateList
+     plural: templates
+     singular: template
+-  scope: Cluster
++  scope: Namespaced
+   versions:
+   - name: v1alpha1
+     schema:
+diff --git a/config/crd/bases/tinkerbell.org_workflows.yaml b/config/crd/bases/tinkerbell.org_workflows.yaml
+index e9a4d5a..11f871d 100644
+--- a/config/crd/bases/tinkerbell.org_workflows.yaml
++++ b/config/crd/bases/tinkerbell.org_workflows.yaml
+@@ -16,7 +16,7 @@ spec:
+     listKind: WorkflowList
+     plural: workflows
+     singular: workflow
+-  scope: Cluster
++  scope: Namespaced
+   versions:
+   - name: v1alpha1
+     schema:
+diff --git a/config/default/manager_image_patch.yaml b/config/default/manager_image_patch.yaml
+index af4861a..2b3b15c 100644
+--- a/config/default/manager_image_patch.yaml
++++ b/config/default/manager_image_patch.yaml
+@@ -8,5 +8,5 @@ spec:
+     spec:
+       containers:
+       # Change the value of image field below to your controller image URL
+-      - image: ghcr.io/tinkerbell/cluster-api-provider-tinkerbell:dev
++      - image: ghcr.io/tinkerbell/cluster-api-provider-tinkerbell-amd64:dev
+         name: manager
+diff --git a/config/default/manager_pull_policy.yaml b/config/default/manager_pull_policy.yaml
+index cd7ae12..74a0879 100644
+--- a/config/default/manager_pull_policy.yaml
++++ b/config/default/manager_pull_policy.yaml
+@@ -8,4 +8,4 @@ spec:
+     spec:
+       containers:
+       - name: manager
+-        imagePullPolicy: IfNotPresent
++        imagePullPolicy: Always
+diff --git a/controllers/base.go b/controllers/base.go
+index 4d33243..774725a 100644
+--- a/controllers/base.go
++++ b/controllers/base.go
+@@ -142,6 +142,7 @@ func (bmrc *baseMachineReconcileContext) releaseHardware() error {
+ 
+ 	namespacedName := types.NamespacedName{
+ 		Name: bmrc.tinkerbellMachine.Spec.HardwareName,
++		Namespace: bmrc.tinkerbellMachine.Namespace,
+ 	}
+ 
+ 	if err := bmrc.client.Get(bmrc.ctx, namespacedName, hardware); err != nil {
+@@ -176,6 +177,7 @@ func (bmrc *baseMachineReconcileContext) deprovisionHardware(hardware *tinkv1.Ha
+ 	bmc := &pbnjv1.BMC{}
+ 	namespacedName := types.NamespacedName{
```

Comment:
I think we need to make the BMC CRD `Namespaced` as well. 
I think the PR does not include that?

Reply:
Ah, good catch. Had that in the back of my mind and forgot to do it